packages/persist/lib/routes/environment/environmentId/connection/connectionId/sync/syncId/job/jobId/deleteRecords.ts

```diff
@@ -43,10 +47,11 @@ const handler = async (req: EndpointRequest<DeleteRecords>, res: EndpointRespons
         syncJobId,
         model,
         records,
-        activityLogId
+        activityLogId,
+        merging
     });
     if (result.isOk()) {
-        res.status(204).send();
+        res.status(200).send({ nextMerging: result.value });
```

Comment:
runner only cares about status > 299 so there is no issue changing the status code and the response content